### PR TITLE
Add some badges

### PR DESCRIPTION
**Pull request by Lauraducky**

### CODE REVIEW

**Comment by Lauraducky** [comment]
image

**Review by Chromadream — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 # GlobalX.ChatBots.Core
 
+<p align="center">
+    <a href="https://www.nuget.org/packages/GlobalX.ChatBots.Core/"><img src="https://flat.badgen.net/nuget/v/globalx.chatbots.core" alt="GlobalX.ChatBots.Core nuget package" /></a>
+    <a href="https://travis-ci.org/GlobalX/GlobalX.ChatBots.Core"><img src="https://flat.badgen.net/travis/GlobalX/GlobalX.ChatBots.Core" alt="Pockybot on Travis CI" /></a>
+    <img src="https://flat.badgen.net/github/commits/globalx/globalx.chatbots.core" alt="commits" />
```
Comment (jimcullenaus) [inline README.md:6]: Is total number of commits really a useful metric to display?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+    <img src="https://flat.badgen.net/github/contributors/globalx/globalx.chatbots.core" alt="contributors" />
```
Comment (jimcullenaus) [inline README.md:7]: Ditto here

File: README.md (hunk 1 above)
Comment (Evangelinexx) [reply README.md:6]: Especially given this package probably won't change that much. Still, I don't think it hurts to have it here.

File: README.md (hunk 2 above)
Comment (Evangelinexx) [reply README.md:7]: Again, I don't think it really matters that much, it's an accurate metric so ¯\\\_(ツ)_/¯

**Review by Evangelinexx — DISMISSED** [review]

**Review by Evangelinexx — APPROVED** [review]

**Review by Chromadream — APPROVED** [review]

**Pull request merged**